File: class10/java/AP1002-MediaVetor.java
import java.util.Scanner;

public class MediaVetor {
    public static void main(String[] args) {
        final int MAX = 5;
        double[] notas = new double[MAX];
        double soma = 0;
        Scanner sc = new Scanner(System.in);
        for (int i = 0; i < MAX; i++) {
            System.out.print("Nota " + (i + 1) + ": ");
            notas[i] = sc.nextDouble();
            soma += notas[i];
        }
        System.out.printf("Media: %.1f%n", soma / MAX);
    }
}

File: class10/python/AP1002-MediaVetor.py
# Media dos elementos de um vetor
MAX = 5
notas = [0.0] * MAX
soma = 0.0
for i in range(MAX):
    notas[i] = float(input(f"Nota {i+1}: "))
    soma += notas[i]
print(f"Media: {soma / MAX:.1f}")
